nix/cloud/actions.nix

```
@@ -0,0 +1,14 @@
+{
+  cell,
+  inputs,
+}: {
+  "bitte/ci-push" = {
+    task = "build";
+    io = ''
+      _lib: github: {
+        #repo: "input-output-hk/bitte"
+        push: #branch: "bitte-tests"
```

Just putting this here as a reminder before merge

```suggestion
        push: {}
```